lte/gateway/c/core/oai/tasks/grpc_service/SpgwServiceImpl.cpp

```diff
@@ -206,7 +206,7 @@ Status SpgwServiceImpl::DeleteBearer(
   itti_msg.no_of_bearers = 1;
   for (int i = 0; i < request->eps_bearer_ids_size() && i < BEARERS_PER_UE;
        i++) {
-    itti_msg.ebi[0] = request->eps_bearer_ids(i);
+    itti_msg.ebi[i] = request->eps_bearer_ids(i);
```

Comment:
@ulaskozat what is the fix here? on `spgw_handle_nw_initiated_bearer_deactv_req` the iteration over `ebi[]` will be bounded to `no_of_bearers` which will always be 1

Reply:
this does not fix the bug. but the size does not have to be 1 in general.

Reply:
we should change `itti_msg.no_of_bearers = 1;` as well.